src/async.rs

```diff
@@ -224,8 +249,29 @@ impl<'a, T> FusedFuture for SendFut<'a, T> {
 pub struct SendSink<'a, T>(SendFut<'a, T>);
 
 impl<'a, T> SendSink<'a, T> {
+    /// See [`Sender::is_disconnected`].
     pub fn is_disconnected(&self) -> bool {
-        self.0.is_terminated()
+        self.0.is_disconnected()
```

Comment:
I changed this from `is_terminated` to `is_disconnected` for consistency.
Going down the stack it seemed to me to be the same anyway.